docs/production/upgrade.md

```diff
@@ -484,6 +484,52 @@ instructions for other supported platforms.
    18.04](#upgrading-from-ubuntu-1604-xenial-to-1804-bionic), so
    that you are running a supported operating system.
 
+### Upgrading from Debian 11 to 12
+
+1. Upgrade your server to the latest `7.x` release.
```

Comment:
(Might need adjustment if we’re not merging this for 7.0.)

Reply:
I think we'll backport this to 7.1 is we don't merge it to 7.0, so it's probably OK.